xmodule/library_content_block.py

```diff
@@ -127,17 +142,6 @@ def completion_mode(cls):  # pylint: disable=no-self-argument
         display_name=_("Library Version"),
         scope=Scope.settings,
     )
-    mode = String(
```

Comment:
Deprecating mode is something I wanted to go back to and clean up. Initially, I was worried about when old olx without the "shuffle" field would be read from storage. I was fearful that it wouldn't be able to parse correctly. After some investigation, however, keeping shuffle's default to be "True" allows for the legacy behavior to be maintained without any effort. I looked into augmenting the "parse_xml" function to see if that could overwrite the "shuffle" value, but that gets messy because if a block has mode set and shuffle = false, what do we rely on as our source of truth. This approach of removal keeps it simply for this stupid.